change/@fluentui-react-avatar-8bdbaa84-a19a-41b4-8e84-12aaaf55bb24.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: consume shared AvatarContext",
```

Comment:
(nit) Update this description now that AvatarContext is implemented in react-avatar

Reply:
Updated in https://github.com/microsoft/fluentui/pull/24807/commits/ef70cc85d7564264a5c0978848ed470996deda60